_sass/components/_accordions.scss

```diff
@@ -28,16 +27,16 @@ $accordion-border: 3px solid $color-gray-lightest;
   }
 
   button[aria-expanded=false] {
-    background-image: url('../images/icons/plus.png');
-    background-image: url('../images/icons/plus.svg');
+    background-image: url('../img/plus.png');
+    background-image: url('../img/plus.svg');
```

Comment:
I think we may need to change this file path back to `images/icons/` for our repo. We have these images in that directory now, so the incorrect file path may be why the images appeared to be missing.